src/main/java/org/broadinstitute/hellbender/utils/LoggingUtils.java

```diff
@@ -87,6 +93,22 @@ private static void setJavaUtilLoggingLevel(final Log.LogLevel verbosity) {
         return javaUtilLevelNamespaceMap.get(picardLevel);
     }
 
+    /**
+     * Send log output to file if file name is not null.
+     *
+     * @param fileName  File path for log output
+     */
+    public static void sendLogToFile(final String fileName) {
+        if (fileName != null) {
+            final LoggerContext loggerContext = (LoggerContext) LogManager.getContext(false);
```

Comment:
`FileAppender.createAppender`,  which is a couple of lines down, creates the file. So, that should not be an issue.